attestation-service/verifier/src/az_snp_vtpm/mod.rs

```diff
@@ -43,6 +44,24 @@ impl AzSnpVtpm {
     }
 }
 
+pub(crate) fn extend_claim_with_tpm_quote(
+    claim: &mut TeeEvidenceParsedClaim,
+    quote: &Quote,
+) -> Result<()> {
+    let Value::Object(ref mut map) = claim else {
+        bail!("failed to extend the claim, not an object");
+    };
+
+    let mut tpm_values = serde_json::Map::new();
+    for (i, pcr) in quote.pcrs_sha256().enumerate() {
+        tpm_values.insert(format!("pcr{:02}", i), Value::String(hex::encode(pcr)));
```

Comment:
this placeholder will set pcrs like
```json
{
    "pcr01": "xxx",
    ...
    "pcr13": "xxx"
}
```

with prefix-ed `0`s. Is this intended?

Reply:
yes, that's intended (.e.g for key sorting)